.eslintrc.js

```diff
@@ -51,5 +51,13 @@ module.exports = {
     "prettier/prettier": "error",
     "no-console": "error",
     "no-await-in-loop": "error",
```

Comment:
A case to allow a non interactive element to be interactive. Though I think there will be others beyond those roles in the future.
https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/no-noninteractive-tabindex.md